nservicebus/upgrades/support-policy.md

```diff
@@ -9,7 +9,7 @@ related:
 
 ## Supported Versions
 
-Supported versions include any major version released within the last 3 years. Within a supported major version, the latest minor version is supported, or any minor version released within the last year.
+Supported versions include any major version released within the last 3 years. Within a supported major version, the latest minor version is supported. For the NServiceBus package only (i.e. excluding downstream packages such as [transports](/nservicebus/transports/) and [persistence](/nservicebus/persistence/)), any minor version released within the last year is supported.
```

Comment:
>. i.e. not just the latest supported minor, but all minors for supported majors. In that sense, not sure what you have under NServiceBus is right.

Sorry, but that isn't true, and not what the current text of the policy says. Its the latest minor, plus any that have been released in the past year.

So, for a major that has been released 2 years ago and hasn't had any minors released in 2 years, only the latest minor is supported.